config/localization.yaml

```diff
@@ -1,9 +1,37 @@
+world_frame: "map"
+odom_frame: "odom"
+rover_frame: "base_link"
+use_odom_frame: false
+
 gps_linearization:
   reference_point_latitude: 42.2
   reference_point_longitude: -83.7
   reference_point_altitude: 224
+  publish_tf: false
+
+global_ekf:
+  # whether to use the Dilution of Precision (DoP) covariance estimate from the GPS message,
+  # or to replace it with the following configured covariance matrix
+  use_gps_dop_covariance: false
+  gps_covariance: [[2, 0, 0],
```

Comment:
how were these chosen?

Reply:
These were chosen by manually tuning in sim until the estimated rover pose was reasonably stable but still responsive. They will almost certainly not work very well on the real rover, but I think we'll just have to tune them while testing IRL. One of my members is also working on measuring GPS noise, so when thats done we can use that data for this as well.

Reply:
That info should be documented in at least one spot, either in the code here or on the Wiki for localization. Rather minimize the lack of context for any values/constants we have.

Reply:
agree with zack, more generally if you're going to need seperate param sets for sim and rover you probably should rename localization.yaml to localization_sim.yaml and then explain that in the wiki and a comment on top the file.

Reply:
Here's a question, should I put all these parameters in `localization.yaml` and then make a `sim_localization.yaml`, or should I leave `localization.yaml` and make `ekf.yaml` and `sim_ekf.yaml` just for sim params? I'm leaning towards putting them together because otherwise localization.yaml will be continue to be almost empty, and it doesn't seem like they need to be separate

Reply:
I think that you should do the latter (ekf.yaml and sim_ekf.yaml) because otherwise you'd need to duplicate parameters in sim_localization.yaml right? Or am I missing something

Reply:
yeah fair enough